src/utils/DependsOnManager.ts

```diff
@@ -36,6 +36,19 @@ export class DependsOnManager {
     $$(this.facet.element).toggleClass('coveo-hidden', !this.parentFacetHasSelectedValues);
   }
 
+  private get dependentFacets() {
+    const allFacets = ComponentsTypes.getAllFacetsInstance(this.facet.root);
+    return allFacets.filter(facet => this.facet.id === facet.options.dependsOn);
+  }
```

Comment:
Would a one-liner be good here, since `allFacets` is only used once?
```ts
private get dependentFacets() {
  return ComponentsTypes.getAllFacetsInstance(this.facet.root).filter(facet => this.facet.id === facet.options.dependsOn);
}
```

Reply:
this is more for readability